### DISCUSSION TITLE
Speicherort Logfile ändern

### DISCUSSION THREAD

**Discussion by lestho** [Schrolli91/BOSWatch#541] [General]
Hai,
ist es möglich, dass BW die Logdateien auf einen anderen Speicher schreibt? Hab am Raspy eine NAS hinzugefügt, wo ich ohne Probleme drauf zugreifen kann, nun möcht ich dass BW die Logfiles dahin speichert, nicht auf die Lokale SD?

Danke und Grsß
Tom

**Comment by Schrolli91** [comment]
siehe boswatch.py: 
```
parser.add_argument("-u", "--usevarlog", help="Use '/var/log/boswatch' for logfiles instead of subdir 'log' in BOSWatch directory", action="store_true")
```

Den Ordner könntest du dann per Softlink auf dein NAS umleiten.

Alternativ hier hardcoded ändern:
https://github.com/Schrolli91/BOSWatch/blob/5ebbaba6f15b1e0fc0e01b26cfeaf22ebaf82252/boswatch.py#L93
